assets/js/shapes.js

```diff
@@ -1,31 +1,124 @@
 function getLine(length) {
-    // TODO - write method definition here
+
+    var line = "";
+    for(var count = 0; count < length; count++){
+        line += "*";
+    }
+    return line;
 }
 
 
 
 function getBox(width, height) {
-    // TODO - write method definition here
+   
```

Comment:
Not bad! Not a huge fan of the implementation though.
Consider code-reuse.

It is less redundant to reuse code that has already been written to perform what you need.
It could be more simply expressed as
```
function getBox(width, height) {
    let output="";
    for(let i=0;i<height;i++) {
        output += getLine(width) + "\n";
    }
    return output;
}
```

Reply:
I definitely know i have this issue when coding. I have been getting better at seeing when I start using redundant coding(it is easy to catch when I start copying and pasting lol). I was not sure how to make a new function in JS to try and stick the redundant code in.